Meadowlark-js/services/meadowlark-aws-lambda/.env.example

```diff
@@ -29,5 +29,5 @@ ES_PASSWORD=admin
 
 
 #### Configurable backend plugin PoC - set to an npm package name
-#BACKEND_PLUGIN_NAME=@edfi/meadowlark-plugin-dynamodb-opensearch
-BACKEND_PLUGIN_NAME=@edfi/meadowlark-plugin-mongodb
+#BACKEND_PLUGIN_NAME=@edfi/meadowlark-aws-dynamodb-opensearch
+BACKEND_PLUGIN_NAME=@edfi/meadowlark-local-mongodb-opensearch
```

Comment:
Like services, moved backend plugins from "packages" directory to parallel "backends" directory, tweaked naming to include hosting location.